packages/terra-functional-testing/lib/config/wdio.conf.js

```diff
@@ -1,7 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+const ip = require('ip');
```

Comment:
Do we have an option to override this default value via an env?

Reply:
I plan to carry these forward when they become more relevant, but I can go ahead and add them in this PR:

https://github.com/cerner/terra-toolkit-boneyard/blob/master/config/wdio/wdio.conf.js#L14-L20

I think there might be some issues with how these are defined today, but I'll do some debugging to check. The base url being routed matters quite a bit, if the internal and external ports are mapped differently I'd be curious if this would work as intended when using `browser.url`

Reply:
>I think there might be some issues with how these are defined today, but I'll do some debugging to check.

What issues are you references? What have you found out?

Reply:
Added the port and host overrides, it's been a little while since I wrote the comment above. I don't actually remember where I was headed with that. I think I may have thought there is a situation where the overrides could conflict but I haven't noticed any issues. 

https://github.com/cerner/terra-toolkit/pull/417/commits/1318f5fe0af7567ddce1fce9b712dbe5bee0da8e